### PR TITLE
adding support for InsertUpdateRelationalTable in silverpop API

### PR DESCRIPTION
**Pull request by kmanzana**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem 'rake'
 
 group :test do
+  gem 'pry'
```
Comment (ryanatwork) [inline Gemfile:6]: I think this should actually go in it's own group development:

```
group :development do
   gem 'pry'
end
```

File: spec/helper.rb [hunk 2]
```diff
@@ -7,6 +7,7 @@
 require 'silverpop'
 require 'rspec'
 require 'webmock/rspec'
+require 'pry'
```
Comment (ryanatwork) [inline spec/helper.rb:10]: Not sure if this is actually needed to be required here.

**Comment by ryanatwork** [comment]
Also looks like the Travis build failed, not really concerned about Ruby 1.9 https://travis-ci.org/upworthy/silverpop/builds/156341123

File: spec/helper.rb (hunk 2 above)
Comment (kmanzana) [reply spec/helper.rb:10]: It has to be required somewhere and I was using it to debug tests.  I can remove it from here if you'd like.  It wouldn't make sense to require it in any of the actual gem files.

**Comment by kmanzana** [comment]
Thanks for getting to this so quickly.

The Ruby 2.1.5 build is failing because of an error with the Bundler version.  I was able to reproduce and using a the newest 1.7.15 version fixed it.  Please add a command to that version's job to use a newer version of Bundler (info to do that here travis-ci/travis-ci#3531).

I would agree that the pre Ruby 2 versions probably don't need to be supported.  I would recommend adding 2.2.5 and 2.3.1 versions and bumping the 2.1.5 to 2.1.9 for the Travis jobs.

**Pull request merged**